packages/zoe/src/zoeService/types.js

```diff
@@ -152,12 +152,13 @@
  * Verify that an alleged Invitation is real, and return the Bundle ID it
  * will use for contract code.
  *
- * @param {ERef<Installation>}
+ * @param {ERef<Installation>} allegedInstallation
  * @returns {Promise<BundleID>}
  */
 
 /**
- * @template {object} [OR=any]
+ * @template {object} [OA=any] offer args
+ * @template {object} [OR=any] offer result
```

Comment:
Yes of course. Do you think we should adopt that style? So far we've been using all caps to denote a generic type. Usually one letter. I'm more inclined to `A` for args and `R` for results.

If we want to change to words, I'd prefer that be done separately in a repo-wide change.

Reply:
OA and OR seem strictly worse than either alternative. When a single letter is used, I much prefer to also have some part of the commentary indicate what it's mnemonic for, as is the case here, but I think I saw cases where that was dropped.  Do some of the formats for specifying typescript not allow room for comments on individual parameters or template values?  If that's the case, I'd prefer clearer names.

It's often the case that the role of the templated parameters is opaque from just the type declarations. If I have to specify them for offers, invitations, or start functions, it's really helpful to get an indication of what the missing declaration is for.

Reply:
I think the reason generics are usually initialisms is to distinguish from types, which are StudlyCaps.

I take the point that it's helpful to have the parameter described, and when using the TS syntax there's no documentation string. So I agree when there are multiple type slots and there isn't a documentation string that we should use the more descriptive name, even it it might be confused with a typedef.

Reply:
Done in https://github.com/Agoric/agoric-sdk/pull/6287